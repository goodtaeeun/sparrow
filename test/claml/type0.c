struct _IO_FILE;
typedef struct _IO_FILE FILE;
extern struct _IO_FILE *stdout;
extern __inline __attribute__((__gnu_inline__)) int
vprintf(const char *__restrict __fmt, __builtin_va_list __arg) {
  return vfprintf(stdout, __fmt, __arg);
}
