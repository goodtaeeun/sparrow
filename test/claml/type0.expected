/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "type0.c"
struct _IO_FILE;
#line 2 "type0.c"
typedef struct _IO_FILE FILE;
#line 3
extern struct _IO_FILE *stdout;
#line 5
__inline extern int vprintf(char const *__fmt, __builtin_va_list __arg)
    __attribute__((__gnu_inline__));
#line 5 "type0.c"
__inline extern int vprintf(char const *__fmt, __builtin_va_list __arg) {
  int __cil_tmp3;

  {
    {
#line 6
      __cil_tmp3 = vfprintf(stdout, __fmt, __arg);
    }
#line 6
    return (__cil_tmp3);
  }
}
